Sources/ProjectSpec/Target.swift

```diff
@@ -158,18 +158,16 @@ extension Target: PathContainer {
 
 extension Target {
 
-    static func resolveMultiplatformTargets(jsonDictionary: JSONDictionary) -> JSONDictionary {
+    static func resolveMultiplatformTargets(jsonDictionary: JSONDictionary) throws -> JSONDictionary {
```

Comment:
ok

Reply:
Hi, I have a question about this. In `SpecValidation` we already have a `Project` instance that has a property platform that is not an array. It is a simple string because `SpecValidation` phase I think arrives after `resolveMultiplatformTargets` so we convert that in multiple target.
So in the `SpecValidation` I don't know if platform field was an array. Is there a way to understand in `SpecValidation` if we have done a resolveMultiplatformTargets?

Reply:
Maybe something like this

```
// validate project dictionary
do {
    try specLoader.validateProjectDictionaryWarnings()
} catch {
    warning("\(error)")
}
```

this is the `execute` before to create the project and run, them, the SpecValidation. I could do something like this but that raises errors. Because the problem is that I need the original json to see if platform was an array.

If I do something in SpecValidation we have already parsed the json into a Project and if platform was an array is has been resolved in multiple targets.

Reply:
Or I can store in the Target a property `generatedMultiTargets: Bool` just to understand if we went in `resolveMultiplatformTargets` and to the error check in `SpecValidation`.

Reply:
Looks good, thanks